docs/reference/patches.rst

```diff
@@ -266,6 +266,13 @@ Files: ``apps/openedx/settings/cms/development.py``, ``apps/openedx/settings/lms
 
 File: ``build/openedx/Dockerfile``
 
+.. patch:: openedx-dockerfile-final
+
+``openedx-dockerfile-final``
+======================
```

Comment:
This is going to cause an error when building docs (`make docs`). The `===` line should be just as long as the title above.